gems/quilt_rails/test/quilt_rails/react_renderable_test.rb

```diff
@@ -4,15 +4,27 @@ class ReactRenderableTest < Minitest::Test
     include Quilt::ReactRenderable
 
     def test_render_react_calls_reverse_proxy_with_server_uri_and_csrf
+      Rails.env.stubs(:test?).returns(false)
```

Comment:
originally had this in the existing error handler, but seemed smarter to just fail fast if it's a test.